### DISCUSSION TITLE
readme file of markdown in github

### DISCUSSION THREAD

**Discussion by adeelzafarpbg** [AammarTufail/python_ka_chilla2023#49] [General]
My questions:
1. How we can make a readme file of our markdown sheet in github?
2. Do we need to make such file in our profile or on PKC2023?

**Comment by Masoodullah909** [comment]
> My questions:
> 
> 1. How we can make a readme file of our markdown sheet in github?
> 2. Do we need to make such file in our profile or on PKC2023?

**Answer 1**
To create a README file for a repository on GitHub:

1. Go to the main page of the repository on GitHub.
2. Click the "Create new file" button.
3. In the "Name your file" field, enter "README.md" (including the quotation marks).
4. Write the content of your README file in the text editor. The README file should use Markdown syntax, which is a way to format text using a combination of plain text and special characters. 
5. When you are finished writing the README file, scroll down to the "Commit new file" section at the bottom of the page.
6. In the "Commit message" field, enter a brief message describing the changes you have made.
7. Click the "Commit new file" button to save your changes.

Your README file will now be visible on the main page of your repository on GitHub.

A README file is a text file that provides information about a project, such as its purpose, how to use it, and any dependencies or requirements. It is typically located in the root directory of a repository and is displayed automatically when someone visits the repository on GitHub.

And Not soo worry about GitHub, we have a session on it in PKC2023
I hope this helps! Let me know if you have any questions.


**Answer 2**
Yes You may ask for all the files to submit at the end in a notebook 
Be mentally prepared for that and keep working and practicing hard.
